the-blue-alliance-ios/View Elements/Settings/SwitchTableViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SwitchTableViewCell.swift
+//  The Blue Alliance
+//
+//  Created by Anas Merbouh on 18-05-20.
+//  Copyright © 2018 The Blue Alliance. All rights reserved.
+//
+
+import UIKit
+
+class SwitchTableViewCell: UITableViewCell {
+    
+    @IBOutlet public weak var titleLabel: UILabel!
```

Comment:
Can we make sure we're using dynamic type and setting colors explicitly? Ex: https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/swift/the-blue-alliance-ios/View%20Elements/Base/ReverseSubtitleTableViewCell.swift#L7